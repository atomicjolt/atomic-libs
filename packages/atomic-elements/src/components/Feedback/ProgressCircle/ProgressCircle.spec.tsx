import { render } from "@testing-library/react";
import { expect, it, describe } from "vitest";
import { ProgressCircle } from ".";

describe("ProgressCircle", () => {
  describe("Snapshots", () => {
    it("should match snapshot", () => {
      const res = render(<ProgressCircle />);
      expect(res).toMatchSnapshot();
    });
  });
});
