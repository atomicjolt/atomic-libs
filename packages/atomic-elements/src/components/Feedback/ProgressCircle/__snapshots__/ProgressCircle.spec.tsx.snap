// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ProgressCircle > Snapshots > should match snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        aria-valuemax="100"
        aria-valuemin="0"
        aria-valuenow="0"
        aria-valuetext="0%"
        class="sc-fAUdSK gXZUlC aje-progresscircle"
        height="60"
        id="react-aria-:r0:"
        role="progressbar"
        viewBox="0 0 44 44"
        width="60"
      >
        <circle
          class="sc-egkSDF ldYkQC"
          cx="22"
          cy="22"
          r="16"
        />
        <circle
          class="sc-gtLWhw gpaGmi"
          cx="22"
          cy="22"
          r="16"
          stroke-width="10"
        />
        <circle
          class="sc-blHHSb kQsIYn"
          cx="22"
          cy="22"
          r="16"
          stroke-dasharray="0 100"
          stroke-dashoffset="25"
          stroke-width="10"
        />
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      aria-valuemax="100"
      aria-valuemin="0"
      aria-valuenow="0"
      aria-valuetext="0%"
      class="sc-fAUdSK gXZUlC aje-progresscircle"
      height="60"
      id="react-aria-:r0:"
      role="progressbar"
      viewBox="0 0 44 44"
      width="60"
    >
      <circle
        class="sc-egkSDF ldYkQC"
        cx="22"
        cy="22"
        r="16"
      />
      <circle
        class="sc-gtLWhw gpaGmi"
        cx="22"
        cy="22"
        r="16"
        stroke-width="10"
      />
      <circle
        class="sc-blHHSb kQsIYn"
        cx="22"
        cy="22"
        r="16"
        stroke-dasharray="0 100"
        stroke-dashoffset="25"
        stroke-width="10"
      />
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
